### ISSUE TITLE
Group tabs

### ISSUE THREAD

**Issue by pdouglass** [fairdk/lcrs#5]
The old LCRS appears to have tabs that can group computers together, but this feature never worked.

It was be very useful if it did work, so that each batch of computers could be grouped together in tabs.

**Comment by benjaoming** [comment]
Yeah, I agree. I like both the idea of groups and being able to name the groups. Like "Right side of table".
